cmd/installer/main.go

```diff
@@ -23,6 +23,7 @@ func parseOptions(uninstall *bool, options *install.SafeOptions) *rest.Config {
 	flag.DurationVar(&options.OptionsCommon.ReconciliationInterval, "reconciliation-interval", constants.DefaultFullSyncInterval, "Interval of reconciliation loop")
 	flag.StringVar(&options.OptionsCommon.Tag, "tag", "latest", "Image tag")
 	flag.StringVar(&options.Etcd.Servers, "etcd-servers", "", "etcd server addresses")
+	flag.BoolVar(&options.SkipLivenessProbes, "skip-liveness-probes", false, "Don't specify a liveness probe on Controller and API server deployments. Can fix an issue in environements where HTTPS livenessprobe fail")
```

Comment:
Can you rephrase the description and remove the `Don't` part but instead `Disable liveness probe ... Use this when HTTPS liveness probe fails.` ?

Reply:
Yeah sorry, the ... mean fill it with the remaining part 😅

Reply:
Disable liveness probe on Controller and API server deployments. Use this when HTTPS liveness probe fails.